docs/content/configuration/upstream.md

```diff
@@ -23,6 +23,6 @@ Additionally in the current or new config file, set the following parameters as
 1. Restart Athens specifying the updated current or new config file.
 
     ```
-    /proxy  -config_file <path-to updated  current or new configfile>
+     ./proxy  -config_file <path-to updated  current or new configfile>
```

Comment:
Instead of `./proxy`, I would probably change it to `<path_to_athens>/proxy`

Reply:
Makes sense. Will update soon.